### PR TITLE
Consistently name event callbacks on_[event]

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

Changes are only made to examples and internal methods, so no deprecation necessary.

### CODE REVIEW

File: lib/matplotlib/animation.py [hunk 1]
```diff
@@ -1251,10 +1251,10 @@ def _setup_blit(self):
         self._blit_cache = dict()
         self._drawn_artists = []
         self._resize_id = self._fig.canvas.mpl_connect('resize_event',
-                                                       self._handle_resize)
+                                                       self._on_resize)
         self._post_draw(None, self._blit)
 
-    def _handle_resize(self, *args):
+    def _on_resize(self, *args):
```
Comment (anntzer) [inline lib/matplotlib/animation.py]: do you want to fix the signature as well?

Comment (timhoffm) [reply lib/matplotlib/animation.py]: Sure.

**Comment by NelleV** [comment]
:+1: 
Thanks @timhoffm !

**Comment by NelleV** [comment]
oups… I meant to press the merge button after I commented. Sorry!

**Pull request merged**